### PR TITLE
Added PR template

### PR DESCRIPTION
**Pull request by JezSw**
Added template for future use.

### CODE REVIEW

**Comment by JezSw** [comment]
Please review @dodu94 and/or @alexvalentine94

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+# Testing
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- Test A
+- Test B
+- etc
+
+**Test Configuration**:
```
Comment (alexvalentine94) [inline .github/PULL_REQUEST_TEMPLATE.md:38]: I would say this additional detail is on the test configuration is not needed.

**Review by alexvalentine94 — CHANGES_REQUESTED** [review]
In general I think this is a good template and fairly complete. It could be shortened slightly- see comment.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (JezSw) [reply .github/PULL_REQUEST_TEMPLATE.md:38]: Removed

**Review by alexvalentine94 — APPROVED** [review]
Thanks Jez, I think this is a good template. We can refine as needed once it starts being used.

**Comment by JezSw** [comment]
Thanks @alexvalentine94, I can't seem to merge this myself. Can you take a look and/or give me the permissions to do so?

**Comment by dodu94** [comment]
I'll merge it due to positive review

**Pull request merged**